booking-engine-api/operations/hotels.md

```diff
@@ -419,11 +423,21 @@ The availability and pricing is returned for each applicable combination of occu
 | `ProductIds` | array of string | optional | Unique identifiers of products which should be included into pricing calculations. |
 | `CurrencyCode` | string | optional | ISO 4217 code of the currency. If specified the prices in response will contain only single currency based on the code provided. [Supported currency codes](../guidelines/supported-currency-codes.md) |
 | `VoucherCode` | string | optional | Voucher code enabling special rate offerings. |
-| `AdultCount` | number | optional | Requested number of adults. If provided together with `ChildCount`, then `RoomOccupancyAvailabilities` will be computed only for that combination instead of all possible. If `RoomCategory` doesn’t support the given values, then the nearest applicable values are found. |
-| `ChildCount` | number | optional | Requested number of children. |
+| `OccupancyData` | array of [Occupancy Data](#occupancy-data) | required | Occupancy numbers for the reservations. |
```

Comment:
@progresak Should we mention that one of them needs to be adult? Do you know? I think we had a limitation like that?

Reply:
API has valid response when at least one OccupancyData is present, does not depend on adult/child.

Reply:
But for the request?